### PR TITLE
GCI-142: Extracted assignments out of expressions

### PR DESCRIPTION
**Pull request by anuar2k**
## Description of what I changed
I've extracted assignments out of expressions.
The issue in `OrderContext` was fixed by another student, unfortunately leaving a doubled return statement, so I've removed it too.

This is one of many replacements to my previous PR, #2760 

## Issue I worked on
see https://issues.openmrs.org/browse/GCI-142

## Checklist: I completed these to help reviewers :)
- [x] My pull request only contains **ONE single commit**
- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.
- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)
- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.
- [x] All new and existing **tests passed**.
- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/OrderContext.java [hunk 1]
```diff
@@ -65,7 +65,6 @@ public void setCareSetting(CareSetting careSetting) {
 	public Map<String, Object> getContextAttributes() {
 		if (contextAttributes == null) {
 			contextAttributes = new HashMap<>();
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/OrderContext.java]: Is this related to the changes you have made in Concept.java?

Comment (anuar2k) [reply api/src/main/java/org/openmrs/api/OrderContext.java]: No. This part of code was partially fixed with #2750 (by splitting up assigment and return statement) but it left us with doubled `return` statement, as you can see, so I've decided to remove a duplicate (this change really adresses the same type of a warning).

Comment (anuar2k) [reply api/src/main/java/org/openmrs/api/OrderContext.java]: I've addressed this issue with #2729 way before but my PR was not merged. Later another student proposed a fix to the same issue, leaving an additional, unecessary statement, which I removed now.

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/api/OrderContext.java]: For unrelated changes, we recommend separate pull requests.

**Comment by anuar2k** [comment]
Closing. Replaced with #2774

**Pull request closed without merging**